pkg/text/text.go

```diff
@@ -0,0 +1,16 @@
+package text
+
+// TODO Text type:
+// TODO func NewText(text string, options ...TextOption) *Text {}
```

Comment:
To simplify, we could call this `New` so it's invoked with `text.New()` outside of this package.